openapi/generator.go

```diff
@@ -758,6 +758,22 @@ func (g *Generator) newSchemaFromStructField(sf reflect.StructField, required bo
 	if t, ok := sf.Tag.Lookup(formatTag); ok {
 		schema.Format = t
 	}
+
+	// Set example value from tag to schema
+	if e := sf.Tag.Get("example"); strings.TrimSpace(e) != "" {
```

Comment:
`strings.TrimSpace` is only used in the condition check here.
Instead, write `strings.TrimSpace(sf.Tag.Get("example"))`.